meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -239,6 +239,7 @@ do_install() {
         install-bin \
         install-identity-scripts \
         install-inventory-scripts \
+        ${@bb.utils.contains('MENDER_FEATURES_ENABLE', 'mender-inventory-geo-script', 'install-inventory-geo-script', '', d)} \
```

Comment:
This doesn't need to be global. Instead, see how `modules` is used in `PACKAGECONFIG`, two lines below.